ui/selectors/multichain.test.ts

```diff
@@ -115,18 +118,29 @@ function getEvmState(chainId: Hex = CHAIN_IDS.MAINNET): TestState {
       conversionRates: {},
       assetsMetadata: {},
       accountsAssets: {},
+      isEvmSelected: false,
+      multichainNetworkConfigurationsByChainId:
+        AVAILABLE_MULTICHAIN_NETWORK_CONFIGURATIONS,
+      selectedMultichainNetworkChainId: BtcScope.Mainnet,
+      bitcoinSupportEnabled: true,
     },
   };
 }
 
-function getNonEvmState(account = MOCK_ACCOUNT_BIP122_P2WPKH): TestState {
+function getNonEvmState(
+  account = MOCK_ACCOUNT_BIP122_P2WPKH,
+  selectedChainId: CaipChainId = BtcScope.Mainnet,
+): TestState {
   return {
     metamask: {
       ...getEvmState().metamask,
       internalAccounts: {
         selectedAccount: account.id,
         accounts: MOCK_ACCOUNTS,
       },
+      // Error: Type 'BtcScope' is not assignable to type 'SupportedCaipChainId'.ts(2322)
+      // @ts-expect-error - unsure why this error occurs since BtcScope.Mainnet is part of SupportedCaipChainId.
```

Comment:
We should use `SupportedCaipChainId` instead here. Given that `selectedMultichainNetworkChainId` is using this type, that's why we are getting a compiler error.

Not all `CaipChainId` can be included in `SupportedCaipChainId`.


```suggestion
  selectedChainId: SupportedCaipChainId = BtcScope.Mainnet,
): TestState {
  return {
    metamask: {
      ...getEvmState().metamask,
      internalAccounts: {
        selectedAccount: account.id,
        accounts: MOCK_ACCOUNTS,
      },
      // Error: Type 'BtcScope' is not assignable to type 'SupportedCaipChainId'.ts(2322)
      // @ts-expect-error - unsure why this error occurs since BtcScope.Mainnet is part of SupportedCaipChainId.
```